content/docs/user-guide/packaging/windows-release-builds.md

```diff
@@ -194,6 +181,27 @@ Use `Visual Studio 16` as the generator for Visual Studio 2019, and `Visual Stud
 
 The result is a project game release layout in the install directory that's located at `<install>\bin\Windows\release\Monolithic`. In your project game release layout, your `engine.pak` file is located in `Cache\pc`.
 
+Your release build is now complete! Next, you need to [bundle your content](#bundle-content).
+
+#### Option 2: Non-monolithic (Game + Editor/Tools)
+
+The other option is to release your game plus the **O3DE Editor** and all of its tools, which is is a **non-monolithic** build. This is not recommended since the size of all these tools combined will be much larger than the **monolithic** build.
+
+A source engine supports non-monolithic projects by default. As detailed in [Prerequisites](#prerequisites), you should have already created a Visual Studio project for your project and registered your project to your engine.
```

Comment:
nit: Seems like a missing word `A source engine (build)`?

Reply:
Good call-out, it's not really a source engine build though. Maybe "Building from a source engine supports non-monolithic projects by default."